bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
 #----alsa install
 sudo apt-get install -y alsa-base alsa-utils
```

Comment:
also, i dont seem to be able to comment on it, but above (lines 11-25), i would make those 3 sections optional and only do them via command line flags, because they're not really necessary to making a functioning tjbot.  we don't need to do any of those on our bots.

- --disable-ipv6
- --enable-google-dns
- --locale=en_US

or something like that.